### PR TITLE
Update emitter

### PR DESCRIPTION
**Pull request by pgiri4**
A new CalPulser data is introduced for emitter simulation. One can now easily mimic the CalPulser scenario in simulation to compare with real data .

Issue being addressed: Adding new feature

### CODE REVIEW

File: NuRadioMC/SignalGen/emitter.py [hunk 1]
```diff
@@ -75,12 +75,14 @@ def get_time_trace(amplitude, N, dt, model, full_output=False, **kwargs):
         sigma = half_width / (np.sqrt(2 * np.log(2)))
         trace = 1 / (sigma * np.sqrt(2 * np.pi)) * np.exp(-1 / 2 * ((time - 500) / sigma) ** 2)
         trace = amplitude * 1 / np.max(np.abs(trace)) * trace
-    elif(model == 'idl1' or model =='hvsp1'):  # the idl & hvsp waveforms gemerated in KU Lab stored in hdf5 file
+    elif(model == 'idl1' or model == 'hvsp1' or model == 'calPulser'):  # the idl & hvsp2 waveforms gemerated in KU Lab stored in hdf5 file
         path = os.path.dirname(os.path.dirname(__file__))
         if(model == 'idl1'):
-            input_file = os.path.join(path, 'data/idl1_data.hdf5')
-        else:
+            input_file = os.path.join(path, 'data/idl_data.hdf5')
```
Comment (anelles) [inline NuRadioMC/SignalGen/emitter.py:81]: Was this intentional to change the file name, but not add a new file?

Comment (anelles) [reply NuRadioMC/SignalGen/emitter.py:81]: This shouldn't work now.

File: NuRadioMC/SignalGen/emitter.py [hunk 2]
```diff
@@ -32,9 +32,9 @@ def get_time_trace(amplitude, N, dt, model, full_output=False, **kwargs):
         * cw : a sinusoidal wave of given frequency
         * square : a rectangular pulse of given amplituede and width
         * tone_burst : a short sine wave pulse of given frequency and desired width
-        * idl & hvsp : these are the waveforms generated in KU lab and stored in hdf5 files
+        * idl1 & hvsp1 : these are the waveforms generated in KU lab and stored in hdf5 files
         * gaussian : represents a gaussian pulse where sigma is defined through the half width at half maximum
-
+        * calPulser : a new normalized voltage signal which depicts the original CalPulser shape used in ARA-02
```
Comment (cg-laser) [inline NuRadioMC/SignalGen/emitter.py:37]: can you choose a more descriptive name such as "ARA02-CalPulser". We will have many CalPulsers in the future

**Review by cg-laser — CHANGES_REQUESTED** [review]
Hey Pawan. Looks good. In addition to Annas comment I have another small request (see below)

File: NuRadioMC/SignalGen/emitter.py (hunk 1 above)
Comment (pgiri4) [reply NuRadioMC/SignalGen/emitter.py:81]: This is a typo, Thanks Anna.

**Review by cg-laser — APPROVED** [review]

**Pull request merged**